src/main/java/com/oltpbenchmark/benchmarks/templated/util/ComplexValue.java

```diff
@@ -0,0 +1,82 @@
+package com.oltpbenchmark.benchmarks.templated.util;
+
+import java.util.Random;
+
+import com.oltpbenchmark.distributions.ScrambledZipfianGenerator;
+import com.oltpbenchmark.distributions.ZipfianGenerator;
+
+public class ComplexValue {
+    String dist;
+    Long min;
+    Long max;
+    Long seed;
+    String value;
+    String minS;
+    String maxS;
+
+    public final Object randGen;
+
+
+    public ComplexValue(String dist, String min, String max, String seed, String value) {
+        this.dist = dist == null ? "" : dist;
+        this.min = min == null ? 0 : Long.parseLong(min);
+        this.max = max == null ? 1 : Long.parseLong(max);
+        this.seed = seed == null ? 0 : Long.parseLong(seed);
+        this.value = value;
+        this.randGen = createGen(dist,this.min,this.max,this.seed);
+        this.minS = min;
+        this.maxS = max;
+    }
+
+    private Object createGen(String distribution, Long min, Long max, Long seed) {
+
+        if(distribution == null || distribution.length() < 1) return null;
+
+        switch (distribution) {
+            case "uniform":
```

Comment:
I think I'd prefer these as enums to ensure that they're consistent. What do you think?